resize.go

```diff
@@ -294,12 +294,14 @@ func Fill(img image.Image, width, height int, anchor Anchor, filter ResampleFilt
 
 	var tmp *image.NRGBA
 	if srcAspectRatio < minAspectRatio {
-		tmp = Resize(img, minW, 0, filter)
+		scale := float64(srcW) / float64(minW)
+		tmp = CropAnchor(img, srcW, int(float64(minH)*scale), anchor)
```

Comment:
I think it's better to round the result of the crop size calculation, e.g.:
 ```go
int(float64(minH)*scale + 0.5)
```

We should also make sure that the crop width/height is at least 1px. As of now it can produce a zero-size image. Example (1x1 -> 10x1):

```go
img := imaging.New(1, 1, color.White)
img = imaging.Fill(img, 10, 1, imaging.Center, imaging.NearestNeighbor)
fmt.Printf("%dx%d\n", img.Bounds().Dx(), img.Bounds().Dy())
```
```
0x0
```

Reply:
Sounds good, I've added these changes